EnglishFilter/sections/whitelist.txt

```diff
@@ -230,6 +230,9 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/60124
 !+ PLATFORM(ext_ff)
 @@||myhdjav.net^$content
+! https://github.com/AdguardTeam/AdguardFilters/issues/60131
+@@||doodstream.com^$font,image,script,stylesheet,xmlhttprequest,other,domain=dood.to|dood.watch
+@@||dood.watch^$subdocument
```

Comment:
Iframe with video doesn't load f.e. in Chrome.

Reply:
I don't see doodstream.com requests.
@zloyden

Reply:
> Iframe with video doesn't load for example in Chrome.

Looks like they have changed something, now I don't see this request too and subs works good with existing rules

Reply:
And now I see this request again
![Screenshot 2020-07-30 at 14 48 46](https://user-images.githubusercontent.com/18536328/88919633-1f37a080-d274-11ea-929b-f6cb1bdb1943.png)

Reply:
I have the same
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/88922622-0978aa00-d279-11ea-942d-06d7fe1ad7c6.png)

</details><br/>